exampleSite/content/table.md

```diff
@@ -0,0 +1,78 @@
++++
+fragment = "table"
+#disabled = false
+date = "2017-10-10"
+lastmod = "2017-10-10"
+weight = 500
+background = "light"
+selfLink = "table-fragment"
+
+title = "Table Fragment"
+subtitle= "Tables are responsive by default"
+
+unimportant = [true, false, false, false, true, false]
```

Comment:
Let's simplify this for the migration as we can't use $.Scratch inside a partial while using page bundles it seems.
So let's remove the unimportant data for now.

Reply:
Alright. I'll work on these changes and fix the content structure.